common/src/Kokkos_ArithTraits.hpp

```diff
@@ -1644,9 +1644,15 @@ struct [[deprecated]] ArithTraits<qd_real> {
 };
 #endif  // HAVE_KOKKOS_QD
 
+}  // namespace KokkosKernels
+
+namespace Kokkos {
+template <typename T>
+using ArithTraits [[deprecated("Use KokkosKernels::ArithTraits instead")]] = ::KokkosKernels::ArithTraits<T>;
+
 namespace Details {
 template <typename T>
-using ArithTraits [[deprecated("Use Kokkos::ArithTraits instead")]] = ::Kokkos::ArithTraits<T>;
+using ArithTraits [[deprecated("Use KokkosKernels::ArithTraits instead")]] = ::KokkosKernels::ArithTraits<T>;
```

Comment:
How long ago did you deprecate this alias?
Is removing it an option?

Reply:
I am talking about the alias in Details:: that was already marked as deprecated

Reply:
What do you mean "before 5.0"?

Reply:
I don't want to remove this until after the last release in the 4.7 series

Reply:
4.7 was the last minor release in the 4.x series

Reply:
Okay so not 4.7.2 right? straight to 5.0.0?

Reply:
4.7.2 would come from 4.7.1 not develop